src/Microsoft.Kusto.ServiceLayer/Scripting/Contracts/ScriptingRequest.cs

```diff
@@ -24,9 +24,9 @@ public class ScriptingParams
         public string ScriptDestination { get; set; }
 
         /// <summary>
-        /// Gets or sets connection string of the target database the scripting operation will run against.
+        /// Gets or sets the target database the scripting operation will run against.
         /// </summary>
-        public string ConnectionString { get; set; }
+        public string DatabaseName { get; set; }
```

Comment:
Changed from ConnectionString to DatabaseName because the connectionString was being used _just_ to find the databaseName